examples/src/tools/google_finance.ts

```diff
@@ -0,0 +1,23 @@
+import { GoogleFinanceAPI } from "@langchain/community/tools/google_finance";
+import { OpenAI } from "@langchain/openai";
+import { initializeAgentExecutorWithOptions } from "langchain/agents";
+
+export async function run() {
+  const model = new OpenAI({
+    temperature: 0,
+    apiKey: process.env.OPENAI_API_KEY,
+  });
+
+  const tools = [new GoogleFinanceAPI()];
+
+  const financeAgent = await initializeAgentExecutorWithOptions(tools, model, {
```

Comment:
Can you use the `createReactAgent` method from `@langchain/langgraph`?

Reply:
Done!